Alerting docs: HA - configure alertmanager to prevent duplicated notifications (#92611)
grafana/grafana: docs/sources/alerting/set-up/configure-high-availability/_index.md

```

For more information on monitoring alerting metrics, refer to [Alerting meta-monitoring](ref:meta-monitoring). For a demo, see [alerting high availability examples using Docker Compose](https://github.com/grafana/alerting-ha-docker-examples/).

> ## Prevent duplicated notifications

Reviewer: ```suggestion
## Prevent duplicate notifications
```

In high-availability mode, each Grafana instance runs its own pre-configured alertmanager to handle alert notifications.

When multiple Grafana instances are running, all alert rules are evaluated on each instance. By default, each instance sends firing alerts to its respective alertmanager. This results in notification handling being duplicated across all running Grafana instances.

Alertmanagers in HA mode communicate with each other to coordinate notification delivery. However, this setup can sometimes lead to duplicated or out-of-order notifications. By design, HA prioritizes sending duplicate notifications over the risk of missing notifications.

> To avoid duplicated notifications, you can configure a shared alertmanager to manage notifications for all Grafana instances. For detailed instructions, refer to [add an external alertmanager](/docs/grafana/<GRAFANA_VERSION>/alerting/set-up/configure-alertmanager/).

Reviewer: ```suggestion
To avoid duplicate notifications, you can configure a shared alertmanager to manage notifications for all Grafana instances. For detailed instructions, refer to [add an external alertmanager](/docs/grafana/<GRAFANA_VERSION>/alerting/set-up/configure-alertmanager/).
```

> To avoid duplicated notifications, you can configure a shared alertmanager to manage notifications for all Grafana instances. For detailed instructions, refer to [add an external alertmanager](/docs/grafana/<GRAFANA_VERSION>/alerting/set-up/configure-alertmanager/).

Reviewer: ```suggestion
To avoid duplicated notifications, you can configure a shared alertmanager to manage notifications for all Grafana instances. For more information, refer to [add an external alertmanager](/docs/grafana/<GRAFANA_VERSION>/alerting/set-up/configure-alertmanager/).
```

Author: addressed by https://github.com/grafana/grafana/pull/92611/commits/25b042887a57f7c34587929bcbb296e02d7136fd
